rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ReleaseSpecTest.java

```diff
@@ -157,38 +159,70 @@ public void should_document_get_releases() throws Exception {
         String accessToken = TestHelper.getAccessToken(mockMvc, testUserId, testUserPassword);
         mockMvc.perform(get("/api/releases")
                 .header("Authorization", "Bearer " + accessToken)
+                .param("page","0")
+                .param("page_entries", "5")
+                .param("sort","name,desc")
                 .accept(MediaTypes.HAL_JSON))
                 .andExpect(status().isOk())
                 .andDo(this.documentationHandler.document(
+                        requestParameters(
+                                parameterWithName("page").description("Page of releases"),
+                                parameterWithName("page_entries").description("Amount of releases per page"),
+                                parameterWithName("sort").description("Defines order of the releases")
```

Comment:
Maybe it would be good to at least link a documentation on how to properly influence the sorting. So what are the possible values of this param.